mantidimaging/core/operations/divide/divide.py

```diff
@@ -14,11 +14,12 @@
 
 
 class DivideFilter(BaseFilter):
-    """Divides the images by a value. That value is usually the pixel value,
-    and can be specified in either microns or cms.
+    """Divides a stack of images by a value. That value can be the pixel size,
```

Comment:
Is this dividing the pixel values?

Reply:
I think this is to scale the attenuation values by the pixel size. I think maybe this should be hidden from the operations window, as it is called automatically after reconstruction.

Reply:
Yes, it hiding it may prevent confusion later on